README.md

```diff
@@ -183,6 +183,31 @@ to a command update to the device.
 
 ![General ](https://raw.github.com/dmoranj/iotagent-node-lib/develop/img/iotAgentLib.png "Architecture Overview")
 
+### The TimeInstant attribute
+
+The IoT Agent manages the existence of a special timestamp attribute in the device to entity mapping: **TimeInstant**. This
+attribute appears in two different parts of the entity:
+
+* As a metadata of an attribute **TimeInstant** indicates the real timestamp where the value for that measure was registered.
+This timestamp will be provided to the IoTAgent in a protocol-specific way (check your particular IoTAgent for the details).
+
+* As an attribute of the entity, **TimeInstant** indicates the timestamp of the last measure received from the device.
+
+If no information about the measure timestamp is received by the IoTAgent, it will use the arrival time of the measure to
+generate a `TimeInstant` for both the entity and the attribute's metadata.
+
+Take into account that:
+* the timestamp of different attributes of the same measure may not be equal.
+* the arrival time and the measure timestamp will not be the same in the general case.
```

Comment:
I don't know if it will be clear with that expression (although it is also unclear with the current one). What I meant with "arrival time" is the time when the measure arrives to the IoT Platorm, as opposed to the time when the measure is read from the sensors in the device (that would be the one I defined as "measure timestamp").